src/Antiforgery/src/ApiCompatBaseline.txt

```diff
@@ -0,0 +1 @@
+Total Issues: 0
```

Comment:
Can we avoid checking in all of these tiny files? E.g. include _one_ `Total Issues: 0` default file plus those containing issues we actually want to ignore.